apps/etterlatte-saksbehandling-ui/client/src/components/person/aktivitet/Aktivitet.tsx

```diff
@@ -8,116 +8,150 @@ import { hentAktivitspliktVurderingForSak } from '~shared/api/aktivitetsplikt'
 import { formaterDato, formaterDatoMedFallback } from '~utils/formatering/dato'
 import Spinner from '~shared/Spinner'
 import { ApiErrorAlert } from '~ErrorBoundary'
+import { AktivitetspliktTidslinje } from '~components/behandling/aktivitetsplikt/AktivitetspliktTidslinje'
+import { hentFamilieOpplysninger } from '~shared/api/pdltjenester'
+import { Familiemedlem } from '~shared/types/familieOpplysninger'
 
-export const Aktivitet = ({ sakResult }: { sakResult: Result<SakMedBehandlinger> }): ReactNode => {
-  const [hentet, hent] = useApiCall(hentAktivitspliktVurderingForSak)
+const velgDoedsdato = (avdoede: Familiemedlem[] | undefined): Date => {
```

Comment:
kan bruke tomt array i stede for undefined, da slipper du å sjekke `!avdoede`

Reply:
Ser at jeg også har sjekket at avdoed ikke er undefined før jeg kaller på velgDoedsdato også 😅